uber/templates/preregistration/form.html

```diff
@@ -119,8 +41,12 @@
         </div>
     </div>
 
-    <div class="form-group" id="badge_types">
-        <label for="badge" class="col-sm-2 control-label">&nbsp;</label>
```

Comment:
Here's where the two rows are defined.

Side note: I spent about 15 minutes trying to get the labels to vertically center.  Vertical centering is notoriously the hardest thing to do in CSS and I completely failed to get it working.  Not a huge deal, but after this is merged, if someone with more CSS experience wants to take a stab at it I'd be grateful :)